### PR TITLE
New package: filteraudio-git-20150310_1

### PR DESCRIPTION
**Pull request by teh-jazzman**
Audio filter library for use with several TOX clients.

### CODE REVIEW

File: srcpkgs/filteraudio-git/template [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+hostmakedepends="git"
+build_style="gnu-makefile"
+wrksrc="filter_audio"
+
+do_fetch() {
+	git clone https://github.com/irungentoo/filter_audio.git
+}
+
+# Development package
+filteraudio-git-devel_package() {
```
Comment (Gottox) [inline srcpkgs/filteraudio-git/template]: please rename the subpackage to filteraudio-devel-git.

File: srcpkgs/filteraudio-git/template [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+short_desc="Lightweight audio filtering library for TOX"
+maintainer="Stefan Mühlinghaus <jazzman@alphabreed.com>"
+license="BSD 3-clause"
+homepage="https://tox.im"
+hostmakedepends="git"
+build_style="gnu-makefile"
+wrksrc="filter_audio"
+
+do_fetch() {
+	git clone https://github.com/irungentoo/filter_audio.git
```
Comment (xtraeme) [inline srcpkgs/filteraudio-git/template]: Use this:

```
git clone https://github.com/irungentoo/filter_audio.git ${wrksrc}
```

and then you can remove `wrksrc`.

File: srcpkgs/filteraudio-git/template [hunk 3]
```diff
@@ -0,0 +1,27 @@
+# Template file for 'filteraudio-git'
+pkgname="filteraudio-git"
+version="20150313"
+revision=1
+short_desc="Lightweight audio filtering library for TOX"
+maintainer="Stefan Mühlinghaus <jazzman@alphabreed.com>"
+license="BSD 3-clause"
```
Comment (xtraeme) [inline srcpkgs/filteraudio-git/template]: 3-clause-BSD instead.

**Comment by xtraeme** [comment]
Any reason to pkg this even if it's not required by tox? I'd like to avoid adding more -git pkgs if possible.

**Comment by teh-jazzman** [comment]
It is required by two TOX clients. I'm not too comfortable with the git-packages myself, but they are not doing proper releases or even proper version numbers. The three TOX clients have the same problem by the way.

Alternatively I could take the precompiled binaries for the clients from https://wiki.tox.im/Binaries and package those. They have toxcore and filteraudio already statically linked in and we would not need the library packages. They only provide two of the three TOX clients available for Linux though and do not say on the website what the current versions are.

**Comment by teh-jazzman** [comment]
Another option would be to put the filteraudio library in the toxcore-git package. That way there would only be one package for the TOX-specific dependencies. As far as I know it is not used for any other software anyway, but of course I cannot be sure about that.

**Comment by Gottox** [comment]
please solve the merge conflicts, will merge afterwards

**Comment by teh-jazzman** [comment]
The conflicts are in common/shlibs. To properly resolve them I would need to merge with the master branch, which by now is quite a bit ahead of this branch. Any tipps on how to do that without spamming hundrets of commits into this pull request?

**Comment by teh-jazzman** [comment]
Nevermind, I got it to work. But git and I will never be friends...

**Comment by Gottox** [comment]
:smile: I learned to love it

**Pull request merged**